_data/faq.yml

```diff
@@ -0,0 +1,38 @@
+    link:
+      text: Read our comparison blog to find out more.
+      url: https://safing.io/blog/2020/01/22/how-the-spn-compares-to-tor/
+  - q: How does the SPN compare to VPNs?
+    a: The VPN technology was built to securely connect two company locations. However, the SPN was developed from the ground up to protect user privacy. As a result the SPN can easily outperform VPNs in every aspect.
+    link:
+      text: Read our blog post to read all about the differences.
+      url: /blog/2020/01/22/how-the-spn-compares-to-vpns/
+  - q: Are all connections routed through the SPN?
+    a: Yes, when enabled the Portmaster routes all Internet connections through the Safing Privacy Network.
+  - q: Does the SPN route all connections individually? So every connection exits on a different node?
+    a: Yes. Every connection is routed separately, giving you a vast amount of identities (IP addresses).
+    link:
+      text: View this explainer image showing a computer connecting to Steam, ProtonMail and Spotify.
+      url: https://safing.io/assets/img/spn-labeled/spn-community.png
+  - q: Does the SPN route connections on a per app basis?
+    a: No, it routes every connection individually. So as an example, when you open several websites in different tabs in your browser, every connection will be calculated individually.
+  - q: Am I totally safe from tracking with the SPN? What do service providers see when connecting through the SPN?
+    a: No. This is just one piece of the puzzle. The SPN only protects your IP-address from exposure, but there are still many other ways services can track and identify you. These include fingerprinting (mostly in browsers), authentication (when logged in to a service), mouse movements (malicious javascripts), etc...
+  - q: Can services still identify me through other methods?
+    a: Yes, this is just one piece of the puzzle. There are multiple ways services track you, like browser fingerprinting, malicious javascripts, etc... And if you log into services, such as your Google account, they will mostly still know it is you, even though you protect your IP address.
+  - q: Does the SPN only protect my browser?
+    a: The SPN reroutes all your computers connections, including those of your browser.
+  - q: Can I safely log in to sites via the SPN? Like my bank account etc?
+    a: Yes. All serious websites use secure https connections, which encrypts the content of your connection (like login credentials) before the Portmaster wraps them in onion encryption. So no ISP architecture or SPN exit node will be able to manipulate that data. It is protected by https all the way to the destination server.
```

Comment:
We could add here that unencrypted connections (ie. without HTTPS) will exit the network at special nodes with extra protection to ensure nothing funky happens.
We are not doing this yet, but will until the end of the year.

Reply:
it's gonna be easy to update them as soon as we do.